### DISCUSSION TITLE
iOS: Load maps from iCloud drive?

### DISCUSSION THREAD

**Discussion by tgerla** [opengisch/QField#3634] [Q&A]
Hi, I'm wondering if it's possible to load maps directly from your iCloud Drive on iOS devices. I've figured out that I can copy a map from my laptop to my iCloud Drive, then go to the phone, open the Files app, and copy the map contents from iCloud Drive to the QField local storage folder, then in QField, open the map from the "QField files directory". But that's a lot of steps. If I could load a map directly from the iCloud Drive (even in a read-only mode) it would save me loads of time. Thanks!

**Comment by jpasner** [comment]
I've been curious about this functionality as well. It seems that the Apple App environment could be easily used to allow for iCloud integration with QField which would be very useful for large maps and potentially large layers. Once the "read only" functionality is developed it could likely be expanded to read and write. Following to see if this gains traction, would love to help!

**Comment by Rickard9** [comment]
It would be awesome to be able to access projects saved on iCloud from an iOS device!
Hope this comes in a future update!
